### PR TITLE
Add lwp-request CmdStager

### PR DESCRIPTION
**Pull request by bcoles**
https://github.com/rapid7/rex-exploitation/pull/22

### CODE REVIEW

File: lib/msf/core/exploit/cmdstager.rb [hunk 1]
```diff
@@ -24,7 +24,8 @@ module Exploit::CmdStager
     :tftp => Rex::Exploitation::CmdStagerTFTP,
     :wget => Rex::Exploitation::CmdStagerWget,
     :curl => Rex::Exploitation::CmdStagerCurl,
-    :fetch => Rex::Exploitation::CmdStagerFetch
+    :fetch => Rex::Exploitation::CmdStagerFetch,
+    :lwprequest => Rex::Exploitation::CmdStagerLwpRequest
```
Comment (wvu-r7) [inline lib/msf/core/exploit/cmdstager.rb]: Can we rename this to `:lwp_request`?

Comment (wvu-r7) [reply lib/msf/core/exploit/cmdstager.rb]: I understand it's `lwp-request`, so hmm. If you drop that char, there's no need to waffle.

**Comment by wvu-r7** [comment]
There's also these, but if you have the symlink, you should have `lwp-request`!

```
wvu@csv5:~$ ll /usr/bin/{GET,POST}
lrwxrwxrwx 1 root root 11 Dec  5  2015 /usr/bin/GET -> lwp-request
lrwxrwxrwx 1 root root 11 Dec  5  2015 /usr/bin/POST -> lwp-request
wvu@csv5:~$
```

File: lib/msf/core/exploit/cmdstager.rb (hunk 1 above)
Comment (bcoles) [reply lib/msf/core/exploit/cmdstager.rb]: I am breaking new ground by introducing a utility whose name contains never before seen elements of ASCII.

**Comment by bcoles** [comment]
> 
> 
> There's also these, but if you have the symlink, you should have `lwp-request`!
> 
> ```
> wvu@csv5:~$ ll /usr/bin/{GET,POST}
> lrwxrwxrwx 1 root root 11 Dec  5  2015 /usr/bin/GET -> lwp-request
> lrwxrwxrwx 1 root root 11 Dec  5  2015 /usr/bin/POST -> lwp-request
> wvu@csv5:~$
> ```

You would hope so. I initially wrote this with the intention of using `GET`.

**Comment by wvu-r7** [comment]
My OS X box doesn't have the links. Also, it uses `LWP-REQUESTS` as the default method: https://github.com/rapid7/rex-exploitation/pull/22#issuecomment-574034536.

**Comment by wvu-r7** [comment]
I'm updating `Gemfile.lock` and `LICENSE_GEMS` while landing.

**Review by wvu — APPROVED** [review]

**Comment by wvu-r7** [comment]
https://github.com/rapid7/metasploit-framework/commit/2a3f7d8b13d8337618d95c93bffa02132df58701

**Comment by wvu-r7** [comment]
# Release Notes

This adds a command stager for binary payloads that utilizes the `lwp-request (-m GET)` command to fetch a payload over HTTP.

**Pull request merged**